src/Futhark/Actions.hs

```diff
@@ -138,6 +139,35 @@ runCC cpath outpath cflags_def ldflags = do
     Right (ExitSuccess, _, _) ->
       return ()
 
+cmdEMCC :: String
+cmdEMCC = fromMaybe "emcc" $ lookup "emcc" unixEnvironment
```

Comment:
What is the purpose of this?  Is there really a convention of using `emcc` as an environment variable?

Reply:
I tried following what was done here
```haskell
cmdCC = fromMaybe "cc" $ lookup "CC" unixEnvironment
```
But I was just blindly copying code. I think you are correct that it doesn't make sense for emcc.

I'll simply change it to 
```
cmdEmcc = "emcc"
```

Reply:
Just hard-code it.  No need for a variable.  `CC` is a decades-old Unix convention.